### PR TITLE
[en] common VB/VBN confusions not found

### PR DESCRIPTION
**Pull request by AzadehSafakish**
New rule group which takes care of phrases like the following (all of which are not detected by LT):

PRESENT
Am I go the right way? -> going
Are his friends leave today? -> leaving
Isn't he visit his mom? -> visiting
Aren't his friends see the show -> seeing

PAST
Were they send via WhatsApp? → sent
Weren't those houses build with the help of a construction company? → built
Weren't they tell over the phone -> told

And so on...

### CODE REVIEW

**Review by udomai — CHANGES_REQUESTED** [review]

File: languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml [hunk 1]
```diff
@@ -48155,7 +48155,163 @@ USA
...
+                </pattern>
+                <message>Tense error: <suggestion><match no="3" postag="VBG" /></suggestion>.</message>
+                <example correction="going">Am I <marker>go</marker> the right way?</example>
+                <example> The issue is you see the stitching.</example>
+                <example> Here is one reference that might help.</example>
+                <example> What we're doing temporarily is they get three tries.</example>
+                <example> Are you hurt?</example>
+                <example> Is the table set?</example>
+                <example> Is the AC set to high?</example>
+                <example> [Typo] Is it suppose to look like that?</example>
```
[the remark this answers is not in the document]
Comment (udomai) [reply languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml:48209]: Is [Typo] intended as a comment? In that case, don't write it like that, examples are very literal. You can append it as a comment directly after `</example><!-- Typo to be found be other rule -->`

**Comment by udomai** [comment]
The first subrule causes false alarms that can be spotted in the rule editor. The way I tested it was: Copy the first subrule into the rule editor. Take all the general antipatterns (APs), i.e. the ones that appear before the first `<rule>`, and copy them into the rule in the rule editor.

Here are some of the sentences the rule editor finds:

<details>
    <summary>Excerpt from the rule editor results</summary>
    
Here is one **option** to restart your workers. → optioning
What I can say is I **have** been unable to trait the few troops I use. → having
Here is one **example**: → exampling

</details>

I've run the entire rulegroup over the nightly diff corpus (first 700,000 sentences), and here is what it found:

[BE_PART_AGREEMENT_SSC_results.txt](https://github.com/languagetool-org/languagetool/files/6213133/BE_PART_AGREEMENT_SSC_results.txt)

There are many false positives. This is a normal step in rule developing: You check for false positives, and you adjust the pattern, or write antipatterns (and exceptions) until it works :)

Please refine the rulegroup and let me know here or on Slack when you're done.

**Comment by AzadehSafakish** [comment]
updated

**Comment by udomai** [comment]
Updated here: https://github.com/languagetool-org/languagetool/pull/4691

**Pull request closed without merging**